tapir/accounts/models.py

```diff
@@ -76,15 +76,23 @@ def check_password(self, raw_password):
         return self.get_ldap().check_password(raw_password)
 
     def has_perm(self, perm, obj=None):
+        if not hasattr(self, "__cached_perms"):
+            self.__cached_perms = {}
```

Comment:
How I understood issue #336 and from what I think, we should use a constant here

Reply:
I don't think so?
That string is not used anywhere else, and refers to a variable that is only used inside that function. 
#336 was about avoid repetition of string litterals.

Reply:
Yes, I thought you wanted to make the (perm-)strings to constants, maybe also to declare them somewhere centralized